legate/core/store.py

```diff
@@ -1225,7 +1225,9 @@ def compute_projection(self, proj_fn=None, launch_ndim=None):
     def find_restrictions(self):
         return self._transform.convert_restrictions(self._storage.restrictions)
 
-    def find_or_create_legion_partition(self, partition, complete=False):
+    def find_or_create_legion_partition(
+        self, partition, complete=False
+    ) -> Any:
```

Comment:
if we could use `legion.legion_logical_partition_t` here, that's the right type.

Reply:
Not presently. That will be better to sort out once any cffi vs cython dust settles.